kfdefs/overlays/osc/osc-cl1/trino-dev/kustomization.yaml

```diff
@@ -3,10 +3,10 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: odh-trino-dev
 resources:
+  - configs
+  - externalsecrets
+  - hive-metastores
   - kfdef.yaml
   - secure-route.yaml
-  - secrets
-  - hive-metastores
-  - configs
 generatorOptions:
```

Comment:
I thought generatorOptions pertained to secretGenerators. Does it also extend to configurations for configmapGenerators? If so it makes sense to keep this, but if not it should be deleted, no?

Reply:
yes, you'll see configmapgenerator add a hash (config-maps are technically supposed to be immutable, as initially intended anyway), we use this option to disable it

Reply:
Thanks for the explanation! In that case ...
/lgtm
/approve